### DISCUSSION TITLE
Improved documentation for quarto users (working directory, `kableExtra`, and warning messages)

### DISCUSSION THREAD

**Discussion by mitchellcameron123** [Pakillo/grateful#49] [Ideas]
Hi, 

First of all, thank you for making this outstanding package!

I am a quarto user and I have found a few issues that could be clarified in the documentation (hence I am putting this through as an idea). Perhaps these are "bugs" so I am happy to put through a bug report as well. 

First, your documentation advises " It is recommended to set `out.dir = getwd()`.". Running quarto render (in the terminal such as for publishing) will lead to an error. Perhaps the documentation should state that `out.dir = tempdir()` is required in this case. Regardless of quarto, I feel this should be clarified as the description in the help file (`?cite_packages`) states that `getwd()` is recommended by then the coded examples state `tempdir()`.

Second, the `kableExtra` package does process the citations. This is a general problem in quarto and citations in tables [here](https://github.com/quartoS-dev/quarto-cli/issues/3340). Perhaps the package documentation could provide a workaround similar too: 

```
#| output: asis
pkgs <- cite_packages(output = "table", tempdir())

kbl(pkgs, booktabs = T, align = "c") %>%
  kable_styling(full_width = T) %>%
  unclass() %>% cat()
```

Third, when a large number of packages are detected, the warning suggests that `options(renv.config.dependencies.limit = Inf)` is used. However, in quarto at least (perhaps other formats), this will lead to an error. The warning can be resolved by setting it to be sufficiently high instead of `inf` such as`options(renv.config.dependencies.limit = 10000)`. 

I hope these points are helpful for quarto users to be aware of in your documentation. I realise these may be niche cases, but additional documentation may save users some time figuring this all out. 

Kind regards,

**Comment by Pakillo** [comment]
Discussion continued here: https://github.com/Pakillo/grateful/issues/50
